smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/HTTPBindingProtocolGenerator.kt

```diff
@@ -484,9 +488,50 @@ abstract class HTTPBindingProtocolGenerator(
         return containedOperations
     }
 
-    abstract override fun generateMessageMarshallable(ctx: ProtocolGenerator.GenerationContext)
```

Comment:
The 4 functions added in below are moved in from `AWSHTTPBindingProtocolGenerator`.